[TECH-1835] Implement changes on choices screen

av-alexistoledo opened the pull request:
![Screenshot 2023-05-26 at 10 20 12](https://github.com/aion-dk/election-verification-site/assets/112619860/e76e885e-bf1f-4c44-82b8-817cc8f4eaaf)

![Screenshot 2023-05-26 at 10 20 26](https://github.com/aion-dk/election-verification-site/assets/112619860/4546da05-03ad-47bf-b06a-18e8e70f1007)

![Screenshot 2023-05-26 at 10 20 40](https://github.com/aion-dk/election-verification-site/assets/112619860/7a16e781-9de0-473e-9682-5d8897d11fba)

av-alexistoledo committed:
choices screen

av-lukas reviewed (commented):
Consider adding component tests to cover these changes.

av-alexistoledo committed:
[TECH-1832] Welcome screen (#28)

![Screenshot 2023-05-26 at 10 49
43](https://github.com/aion-dk/election-verification-site/assets/112619860/b651ddaf-444e-4aba-a666-92fa6ce28d2e)

![Screenshot 2023-05-26 at 10 49
56](https://github.com/aion-dk/election-verification-site/assets/112619860/e1ab620f-7b5a-46b4-9975-a30473d39237)

![Screenshot 2023-05-26 at 10 50
12](https://github.com/aion-dk/election-verification-site/assets/112619860/f94ce909-1a26-4568-be90-1f2e9179f7ce)
